aspnet/fundamentals/servers.rst

```diff
@@ -95,55 +100,32 @@ ASP.NET defines a number of :doc:`request-features`. The following table lists t
     - Yes
     - No
 
-Configuration options
-^^^^^^^^^^^^^^^^^^^^^
-
-You can provide configuration options (by command line parameters or a configuration file) that are read on server startup.
-
-The ``Microsoft.AspNetCore.Hosting`` command supports server parameters (such as ``Kestrel`` or ``WebListener``) and a ``server.urls`` configuration key. The ``server.urls`` configuration key is a semicolon-separated list of URL prefixes that the server should handle.
-
-The *project.json* file shown above demonstrates how to pass the ``server.urls`` parameter directly:
-
-.. code-block:: javascript
-
-  "web": "Microsoft.AspNetCore.Kestrel --server.urls http://localhost:5004"
+Port 0 binding with Kestrel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Alternately, a  JSON configuration file can be used,
+The :dn:interface:`~Microsoft.AspNetCore.Builder.IApplicationBuilder` available in the ``Startup``'s ``Configure`` method exposes ``ServerFeatures`` of type :dn:interface:`~Microsoft.AspNetCore.Http.Features.IFeatureCollection`. Kestrel and WebListener both expose only a single :dn:interface:`~Microsoft.AspNetCore.Hosting.Server.Features.IServerAddressesFeature`, but different server implementations may expose additional functionality. 
```

Comment:
I just mean that ServerFeatures should get its own section followed by one for Kestrel.